src/inmanta/app.py

```diff
@@ -324,6 +324,32 @@ def help_command(options: argparse.Namespace) -> None:
     sys.exit(0)
 
 
+@command("show-default-logger-config", help_msg="Display the default logger configuration")
+def show_default_logger_config(options: argparse.Namespace) -> None:
+    default_config = """
```

Comment:
The idea was to generate this config dynamically based on the `LoggingConfigBuilder` class. Otherwise there is the risk that this hardcoded default config will over time become out of sync with the config generated by the `LoggingConfigBuilder` class.

Reply:
I won't have time to do this ticket, I'll put it back in the next column.